### PR TITLE
Minigame solvers

### PR DESCRIPTION
**Pull request by Linkis20**
Doing tasks in Airship Map.
Currently implemented:
- ~~Stage 2 of Upload Data Airship~~
- Door Sabotage Airship
- Radio Frequency (Skeld, Polus, Airship)

### CODE REVIEW

**Comment by Linkis20** [comment]
[EBro912](https://github.com/EBro912) commits are in here, but that is because I stated with https://github.com/VedalAI/neuro-amongus/pull/33 branch.

**Comment by Linkis20** [comment]
Made https://github.com/Linkis20/neuro-amongus/tree/minigame-solvers-clean if you need one without [EBro912](https://github.com/EBro912) commits.

File: Neuro/Minigames/Solvers/DoorSabotageAirshipSolver.cs
Comment (Alexejhero) [inline Neuro/Minigames/Solvers/DoorSabotageAirshipSolver.cs:1]: The cursor isn't visible from the start of the movement, you call InsertCard before showing the cursor

File: Neuro/Minigames/Solvers/UploadDataAirshipSolver.cs
Comment (Alexejhero) [inline Neuro/Minigames/Solvers/UploadDataAirshipSolver.cs:1]: Neuro isn't psychic, she shouldn't know where the hotspot is like that.

Implement some sort of scanning motion that scans the whole screen area until it gets to Poor, and then starts scanning a smaller area until it gets to Good, and so on until it goes to Perfect.

File: Neuro/Minigames/Solvers/TuneRadioSabotageSolver.cs
Comment (Alexejhero) [inline Neuro/Minigames/Solvers/TuneRadioSabotageSolver.cs:1]: This solver looks a bit janky, the dial is turned in the correct direction always, the movement is very slow and once it reaches the target, it jitters back and forth.

**Review by Alexejhero — CHANGES_REQUESTED** [review]
Requesting changes for all 3 solvers, if you do not want to change some of them feel free to remove them from the PR and someone else will take them

**Comment by Alexejhero** [comment]
Someone else is working on a solver for the Upload Data Airship task

**Review by Alexejhero — APPROVED** [review]

**Pull request merged**